documentation/src/main/asciidoc/user_guide/cross_site_replication.adoc

```diff
@@ -213,6 +213,47 @@ expired in clusters on other sites.
 {brandname} cannot determine if cache entries have reached the idle timeout in
 clusters on other sites.
 
+[[xsite_concurrent_writes]]
+==== Deadlocks and Conflicting Entries
+In active/active configurations, concurrent writes to the same key results in
+deadlocks or conflicts.
+
+Active/active configurations are those in which both the local and the backup
+sites replicate data to each other. For example, the site "LON" backs up to the
+site "SFO" and "SFO" also backs up to "LON".
+
+The following example illustrates how deadlocks and conflicts occur with
+concurrent writes in active/active configurations:
+
+[source,txt,options="nowrap"]
+----
+include::xsite_conflicts.txt[]
+----
+
+<1> `x` represents a key in the data store.
+<2> client writes `x=3` on "LON". "LON" replicates to "SFO".
+<3> client writes `x=5` on "SFO". "SFO" replicates to "LON"
+<4> client writes `x=6` on "LON" at the same time that a client writes `x=8` on "SFO".
+
+In synchronous mode (`SYNC`), the preceding example results in a deadlock where
+{brandname} clusters in both "LON" and "SFO" lock `x`. To resolve deadlocks,
+client applications must wait until the locks time out.
+
+In asynchronous mode (`ASYNC`), the preceding example results in conflicts
+with one of the following outcomes:
+
+* `x` has a different value on "LON" than on "SFO". Asynchronous replication
+occurs after the client writes to `x` with the result that `x=8` is replicated
+to "LON" and `x=6` is replicated to "SFO".
+* `x` has the same value on "LON" and "SFO" because asynchronous replication
+overwrites one of the values, depending on when the transfer occurs. In this
+case, one of the values is lost and there is no guarantee which value is saved.
+
```

Comment:
I would mention that the inconsistence will be solved with the next non conflicting put